client/README.md

```diff
@@ -128,98 +128,35 @@ Or, if you want to persist data across restarts, you will need to run the client
 ./run.sh --sdc-home /path/to/my/configuration/directory
 ```
 
-### Developer environment on a Linux-based non-Qubes OS
```

Comment:
Somehow this ended up including helpful pointers that we didn't give to developers on other operating systems, even though they were generally applicable. Apparently Mac users get all the love. ;-) I tried to consolidate this a bit more since there isn't really too much Mac-specific stuff in here.